doc/ref_arch/openstack/chapters/chapter08.md

```diff
@@ -5,7 +5,28 @@
 
 ## Table of Contents
```

Comment:
@sukhdevkapur please seperate ToC from the rest of the document. look at examples from others chapters. (for example: https://raw.githubusercontent.com/cntt-n/CNTT/master/doc/ref_model/chapters/chapter01.md)